Webhook/WAWebhookNew.js

```diff
@@ -146,6 +147,8 @@ const handleTextMessage = async (message, from, phone_number_id) => {
             sendWatsAppWithList(response.answer, options, 'How can I help you Today?', 'EqualJustice.ai', from, phone_number_id);
             return;
         }
+    } else {
+        session.interactions++;
```

Comment:
only increasing counter in session object will not work, you need to update it in Radis session using save session function

Reply:
when you want to update something in session storage, you need to save it in radis, check the saveSessionforPayment() function you need to write similar function to update the interactions counter in save session